loadtestconfig.rst

```diff
@@ -0,0 +1,571 @@
+========================
+Load Test Configuration
+========================
+
+The following configurable variables comprise the loadtestconfig.json. 
+It is part of the ``mattermost-load-test-master`` directory at the top level. 
```

Comment:
Master? It's just mattermost-load-test. But I think this is unnecessarily verbose.